sensor_msgs/PointCloud2 cloud
sensor_msgs/PointCloud2 redundancy_cloud
---
bool success